extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/AssetMapper.java

```diff
@@ -0,0 +1,49 @@
+package de.sovity.edc.ext.wrapper.api.common.mappers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.sovity.edc.ext.wrapper.api.common.model.UiAsset;
+import de.sovity.edc.ext.wrapper.api.common.model.UiAssetHelperDto;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+
+
+@RequiredArgsConstructor
+public class AssetMapper {
+    /**
+     * This Object Mapper must be able to handle JSON-LD serialization / deserialization.
+     */
+    private final ObjectMapper jsonLdObjectMapper;
+
+
+    public UiAsset buildUiAsset(UiAssetHelperDto uiAssetHelperDto) {
```

Comment:
both these functions should be private. the API of this class will be used directly in multiple places (remember, common-mappers is a library of sorts to be re-used in the broker, too)

https://github.com/sovity/edc-extensions/blob/9646d08eaae7f47bc42eb3ba9f27382ca9f81ef6/extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/AssetMapper.java